### DISCUSSION TITLE
Autoload Repeater by default with all employee data

### DISCUSSION THREAD

**Discussion by jawadsiddiqui** [filamentphp/filament#12135] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

I have payrun filament resource form where I added this repeater below

`
Repeater::make('employeeData')
                    ->relationship()
                    ->schema([
                        
                    ])
                 ->columns(2)`
I need this repeater load with all employees mean if I have 10 employee in it show 10 rows in repeater with employee Code thats it

**Comment by leandrocfe** [comment]
Do you have a hasMany relationship?
https://filamentphp.com/docs/3.x/forms/fields/repeater#integrating-with-an-eloquent-relationship

**Reply by jawadsiddiqui** [reply]
Let me explain payroll_run is schema where I initially want the salary_data column of datatype JSON 

I want salary_data: {employee_id, salary, deduction, etc....} to be saved this is possible with a repeater but want that repeater to load all employee rows. when I click on create. tell me if there is any relevant solution
